doc/md_src_files/082_relatedSystems.md

```diff
@@ -105,3 +105,74 @@ In that sense, it is compatible with any of methods mentioned above or SSVC.
+Once that list were curated, it could be used to automatically populate a CVE-ID as [*proof of concept*](#exploitation) if the CWE-ID of which it is an instance is on the list.
+Such a check could not be exhaustive, since there are other conditions that satisfy [*proof of concept*](#exploitation), but if paired with automatic searches for exploit code in public repositories, it would cover many scenarios.
+If paired with active exploitation feeds discussed above, then the value of  [*Exploitation*](#exploitation) could be determined almost entirely from available information without direct analyst involvement at each organization.  
+
+[CPE](https://cpe.mitre.org/specification/) could possibly be curated into a list of representative [*Public Safety Impact*](#public-safety-impact) values for each platform or product.
+The [*Situated Safety Impact*](#situated-safety-impact) would be too specific for a classification as broad as CPE.
+But it might work for [*Public Safety Impact*](#public-safety-impact), since it is concerned with a more general assessment of usual use of a component.
+Creatinga mapping between CPE and [*Public Safety Impact*](#public-safety-impact) could be a community effort to associate a value with each CPE entry, or an organization might label a fragment of the CPE data with [*Public Safety Impact*](#public-safety-impact) based on the platforms that the supplier needs information about most often.
+
+## Potential Future Information Feeds
+
+So far, we have identified information sources that can support scalable decision making for most decision points.
+Some sources, such as CWE or existing asset management solutions, would require a little bit of connective glue to support SSVC, but not too much.
+The SSVC decision point that we have not identified an information source for is [*Utility*](#utility).
+[*Utility*](#utility) is composed of [*Automatable*](#automatable) and [*Value Density*](#value-density), so the question is what a sort of feed could support each of those decision points.
+
+These are both decision points where a feed is plausible.
+The values for [*Automatable*](#automatable) and [*Value Density*](#value-density) are both about the relationship between a vulnerability, the attacker community, and the aggregate state of systems connected to the Internet.
+While that is broad analysis frame, it means that any community that shares a similar set of adversaries and a similar region of the Internet can share the same response to both decision points.
+An organization in the People's Republic of China may have a different view than an organization in the United States, but most organizations within each region should should have close enough to the same view to share values for [*Automatable*](#automatable) and [*Value Density*](#value-density).
+These factors suggest there would be a viable market for an information feed about these decision points.
```

Comment:
s/would/could/

Reply:
I think that changes the meaning too much, consider the pushed change?

Reply:
Yeah, I was mainly concerned about asserting the probable future existence of a viable market versus the plausibility of such a market.